src/Native/System.Security.Cryptography.Native/pal_eckey.cpp

```diff
@@ -12,33 +12,59 @@ extern "C" void CryptoNative_EcKeyDestroy(EC_KEY* r)
     EC_KEY_free(r);
 }
 
-extern "C" EC_KEY* CryptoNative_EcKeyCreateByCurveName(int32_t nid)
```

Comment:
The problem would come about if someone has

System.Security.Cryptography.OpenSsl-4.0.0-rc2-whateverthefinalrc2buildis (maybe because they had X509Certificates from rc2 netstandard)
System.Security.Cryptography.Algorithms-4.1.0-rc3-somefuturerc3build

The newest CryptoNative will be selected, and it'll count as a met dependency for the OpenSsl library.  Likewise, the Algorithms library will count as a met dependency for the OpenSsl library.  But that OpenSsl library is looking for the old native export, which no longer exists.